src/opencmiss/zincwidgets/__init__.py

```diff
@@ -1,3 +1,3 @@
 
 # OpenCMISS Zinc widgets Python package initialisation file.
-__version__ = "2.0.3.dev1"
+__version__ = "2.0.4.dev1"
```

Comment:
We can just update the `dev1` to `dev2` here.

Reply:
Fixed.